ansible/deploy_proxy.yml

```diff
@@ -70,7 +102,7 @@
         hosts_group: "{{ 'riakcs_new' if 'BLOB_DB_MIGRATING_FROM_S3_TO_S3' in localsettings else 'riakcs' }}"
         riakcs_proxy_port: "{{ temp_riakcs_proxy_port }}"
         site_config: {name: riakcs, vars_file: riakcs}
-      when: "active_sites.riakcs == True
+      when: "groups.get('riakcs')
```

Comment:
should this not also mimic the `riakcs site` task above and use `set_site_present`?

Reply:
Ah! I think you're right. I originally decided that since I didn't understand the details behind this, I just wouldn't change the current behavior. However, I'm just noticing that the following task ("drop riak temp") has exactly the opposite logic (okay well, I think `groups.get('riakcs') and` should actually be `not groups.get('riakcs') or (...)`, in which case it _would_ be the exact opposite), so it's pretty clearly what's intended.

I'll switch this and the aforementioned following task to use `set_site_present` in a followup PR. Thanks for pointing out!